server/assets/asset.go.bak

```diff
@@ -0,0 +1,40 @@
+// satisfies go-bindata interface
```

Comment:
bak file in the repository looks weird...

Reply:
I'm open to suggestions ;)
The point is that I think we should ensure we do not destroy this file... but how to do it... 
ideas? :dancer:

Reply:
For this PR it's maybe alright. But I would like to re-think how we work with assets. Maybe we can find better solution.

Reply:
^ that's a great idea. I opened an issue https://github.com/src-d/gitbase-playground/issues/24 to avoid losing your valuable suggestion.